### PR TITLE
HTM-24 create edit facilities

### PR DESCRIPTION
**Pull request by rafi-alam**

### CODE REVIEW

**Comment by shakil-muntasir** [comment]
Fix merge conflict @rafi-alam

**Comment by shakil-muntasir** [comment]
There are some fixes you have to apply. Check the screenshot

image

**Comment by rafi-alam** [comment]
@shakil-muntasir conflict fixed

**Comment by shakil-muntasir** [comment]
> 

Did you realize you just removed expected changes from App.vue and called it "conflict fixed"?
The Navigation is no longer available in App.vue!
I have also instructed you to use `git rebase origin/main`
Watch a video if you have to. Fix this again...

File: resources/js/router/index.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-import Register from '../pages/auth/Register.vue'
+import Facilities from '../pages/Facilities.vue'
+import Form from '../components/facilities/Form.vue'
 
 const routes: RouteRecordRaw[] = [
     { path: '/', component: Home },
     { path: '/about', component: About },
-    { path: '/register', component: Register }
+    { path: '/facilities', component: Facilities, children: [
+        { path: '/facilities/create', component: Form }
```
Comment (shakil-muntasir) [inline resources/js/router/index.ts]: I think this is wrong...
Shouldn't it be `{ path: '/create', component: Form }` ???

**Comment by shakil-muntasir** [comment]
Looking good finally 😸 Merging

**Pull request merged**